superset-frontend/src/visualizations/FilterBox/controlPanel.jsx

```diff
@@ -66,6 +66,28 @@ export default {
             },
           },
         ],
+        [
+          {
+            name: 'show_sqla_time_granularity',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Show time grain dropdown'),
+              default: false,
+              description: t('Check to include time grain dropdown'),
```

Comment:
Per [here](https://github.com/apache/superset/pull/19770/files#diff-d127efd58418777f8866e21aab14d19c536bba2f639198f2777f260c4c3602b4L104-L106) I think the old labels where mixed, i.e., if Druid was enabled it would include the term SQL. I opted for the clearer labels/description.